lib/setup-tofu.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * Copyright (c) OpenTofu
+ * SPDX-License-Identifier: MPL-2.0
```

Comment:
I like this approach of doubling-up on the copyright holder names — should this be standardized across all files where this notice is present?

Reply:
Please create a ticket to followup on it. I suggest to release v0.0.1 of setup-tofu first. We can reside on headers afterwards and implement that decision.